compose/aspnet-mssql-compose.md

```diff
@@ -0,0 +1,142 @@
+---
+description: Create a Docker Compose application using ASP.NET Core and SQL Server on Linux in Docker. 
+keywords: dotnet, .NET, Core, example, ASP.NET Core, SQL Server, mssql
+title: "Quickstart: Compose and ASP.NET Core with SQL Server"
+---
+
+This quick-start guide demonstrates how to use Docker Compose to set up and run the sample ASP.NET Core application using the [ASP.NET Core Build image](https://hub.docker.com/r/microsoft/aspnetcore-build/) with the [SQL Server on Linux image](https://hub.docker.com/r/microsoft/mssql-server-linux/). You just need to have [Docker Engine](https://docs.docker.com/engine/installation/) and [Docker Compose](https://docs.docker.com/compose/install/) installed on your platform of choice.
+
```

Comment:
Can we put 'Linux' somewhere in the first paragraph, to make it clear that this isn't a Windows container demo?